src/commands/e2e/WdioConfigTemplate.ts

```diff
@@ -157,6 +158,7 @@ exports.config = {
                 browserName: '${this.browser}',
                 'moz:firefoxOptions': {
                     'args': ['-headless']
+                    w3c: false,
```

Comment:
Is Internet Explorer affected as well?

Reply:
yes all browser are affected by this. This change works for chrome but not for the other browser

Reply:
do we need it then for firefox?

Reply:
No, I need to update this again or find another solution for this